src/main/java/mate/academy/Main.java

```diff
@@ -1,11 +1,15 @@
 package mate.academy;
 
+import mate.academy.lib.Injector;
 import mate.academy.model.Movie;
 import mate.academy.service.MovieService;
 
 public class Main {
+
+    private static final Injector INJECTOR = Injector.getInstance("mate.academy");
+
     public static void main(String[] args) {
-        MovieService movieService = null;
+        MovieService movieService = (MovieService) INJECTOR.getInstance(MovieService.class);
```

Comment:
Test your solution! Let’s  add some movie sessions and find available;)

Reply:
not addressed